RNTester/js/TextInputExample.js

```diff
@@ -0,0 +1,258 @@
+/**
```

Comment:
It might make more sense to name file TextInputExample.ubuntu.js since TextInput implementation for desktop differs from implementation for mobile.
Also copyright might be replaced for Status.